### ISSUE TITLE
Create new home page with stats, resume reading links, news and other info

### ISSUE THREAD

**Issue by M-Biggles** [LuteOrg/lute-v3#185]
This is a bit of a bigger feature. It would be nice to have a home page with all kinda of useful stuff on it. Basic stats for each language (known words, words read, hours listened, etc), links to resume reading, Lute news, etc.

The current home page with the active texts could be moved to 'Books', which would be a bit more consistent anyway.

**Comment by jzohrab** [comment]
Right, I can see how that would be nice for some people.  I did the listing just because I didn't want to have to click again to get into what I was reading, I wanted Lute to stay out of the way. :-)  But some might prefer your idea as a starting point.

**Comment by M-Biggles** [comment]
Yeah, absolutely agree that the software needs to stay out of the way. I thought I'd throw this one out there to see what you think.

Maybe if a resume reading section on the home page could be done right it would save you that click. Most recent 5 items or something...

**Comment by jzohrab** [comment]
Ah right, perhaps the book table should sort with the most recently read thing right at the top.  I only keep a few books on the active listing (home page) at a time, and archive when I'm done.  I may be the exception though.

There's a stats page already, but it only has word count read.  There's already another issue for including some kind of word count/breakdown.

**Comment by M-Biggles** [comment]
I know I tend to have a bunch of texts in my library, and I'm not the only one! haha At the moment, since there are no folders / collections for texts, the listings are going to be longer on average. Still, I'll be adding books with 50+ chapters at various points, as well as podcasts with transcripts which number into the hundreds of episodes.

Another additional idea is to allow the user to pin texts to the top.

And, if you don't want to change anything to your own setup, make an option for choosing the default Home Page upon loading Lute -one could be 'welcome-page' and the other 'active-texts' (the current home page - whatever it's called).

**Comment by jzohrab** [comment]
Yes, an option sounds like the right thing.

(There's an open issue for some kind of way of grouping texts -- either folders, or the idea of "shelves" like LingQ has.)

Pinning sounds interesting.  So many different ways to cut the data.  👍

**Comment by GrimPixel** [comment]
Maybe the home page can look like [Anori](https://github.com/OlegWock/anori)?
